### ISSUE TITLE
[Bug]: Redis doesn't run on startup

### ISSUE THREAD

**Issue by Benisy** [beyondcode/herd-community#814]
### Platform

Windows

### Operating system version

Windows 11 Pro 23H2 22631.3737

### System architecture

Windows

### Herd Version

1.7.1

### PHP Version

8.3.8

### Bug description

Redis doesn't run when starting Laravel Heard. If you change the port and restart, everything works.

### Relevant log output

```shell
[009304] 17 Jun 10:39:27.687 * RedisWin is starting ......
[009304] 17 Jun 10:39:27.687 * RedisWin version=7.2.4, bits=64, commit=14/01/2024, modified=0, pid=9304, just started
[009304] 17 Jun 10:39:27.687 * Configuration loaded
[009304] 17 Jun 10:39:27.688 * monotonic clock: X64 clock_gettime
[009304] 17 Jun 10:39:27.688 * Running mode=standalone, port=6379.
[009304] 17 Jun 10:39:27.689 # Warning: Could not create server TCP listening socket *:6379: bind: No such file or directory
[009304] 17 Jun 10:39:27.689 # Failed listening on port 6379 (tcp), aborting.
```

**Comment by sschlein** [comment]
Please use stop all and start all services on your dashboard, we just fixed that and will release an update soon.

**Comment by sschlein** [comment]
Fixed in 1.7.2
